e2e/atlas/access_lists_test.go

```diff
@@ -156,5 +155,19 @@ func TestAccessList(t *testing.T) {
 		req.NoError(err)
 
 		a.NotEmpty(entries.Results)
+
+		cmd = exec.Command(cliPath,
+			atlasEntity,
+			accessListEntity,
+			"delete",
+			entry,
```

Comment:
[q] isn’t this defined in another test? I think you are not deleting `currentIp`

Reply:
Could you also update the above test to run the delete command instead only the create? keep in mind that we should keep using` "--deleteAfter="+time.Now().Add(time.Minute*time.Duration(5)).Format(time.RFC3339),` in the create command so that we know that the flag works as expected.  You just need to define two new test scenarios `t.Run("Delete accesslist entry")`  and add then one after `Create Delete After` and one after you test using `currentIp` as entry

Reply:
q1) you're right I'll need to retrieve this from the result

comment -> let me sync w/ you offline to understand a bit more!